app/models/manageiq/providers/ansible_tower/shared/inventory/parser/automation_manager.rb

```diff
@@ -65,6 +65,10 @@ def configuration_script_sources
       inventory_object.status = project.status
       inventory_object.last_updated_on = project.last_updated
 
+      if project.status == 'failed'
```

Comment:
Good point! I’ll update `last_update_stdout` every time. Since the output capture is presented on a separate subpage, it does not disturb even for the success. Still, it can contain some useful information like warnings and such.